go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -26,6 +26,80 @@ type lbEndpoints struct {
 	Port int32
 }
 
+func getLocalLbEndpoints(slices []*discovery.EndpointSlice, svcPort v1.ServicePort, family v1.IPFamily, gatewayRouter string) lbEndpoints {
```

Comment:
Can you change `getLbEndpoints` to accept an optional filter function that skips?

```
func GetLbEndpoints(slices []*discovery.EndpointSlice, svcPort kapi.ServicePort, family kapi.IPFamily, filter ...func(discovery.Endpoint) bool) LbEndpoints {
```

Reply:
ACK, but still need to address